app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1><%= @recipe.name %></h1>
+<p>Complexity: <%= @recipe.complexity %></p>
+<p>Genre: <%= @recipe.genre %></p>
+
+<h2>Ingredients List</h2>
+<% @recipe_ingredients.each do |ingred| %>
+  <p><%= ingred.name %></p>
+<% end %>
+<h3>Total Cost of Ingredients: <%= @total_cost %></h3>
+
+<%= form_with url: "/recipes/#{@recipe.id}", method: :patch, local: true do |f| %>
```

Comment:
Looks like you were working on the 2nd extension. This is a good start, but I'm not sure we're really "updating a specific recipe" - the user story asks us to make a new relationship between an existing recipe (@recipe.id) and and the _id of an ingredient_ that the user enters in the form. So, we wouldn't need a `:name` field in the form, rather an `:ingredient_id` field. 

Also, in terms of a restful route, our URL should reflect that wer'e adding an ingredient to a recipe; so one of these options: 
```
"/recipes/:id/ingredients"
"/recipe_ingredients"
```
The 2nd option would mean we'd need a hidden field on the form to send in the recipe_id as well as the ingredient_id, but eitehr way would work. 

Try finishing this out for good practice!